.github/workflows/ping.yaml

```
@@ -0,0 +1,15 @@
+name: Keep Render Alive (${{ secrets.PING_ENABLED == 'true' && 'ENABLED' || 'DISABLED' }})
```

actionlint (1.7.7)

context "secrets" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)